EpiAware/docs/examples/toy_model_log_infs_RW.jl

```diff
@@ -143,7 +141,8 @@ truth_data = random_epidemic.y_t
 model = make_epi_aware(truth_data, time_horizon, ; epi_model = epi_model,
     latent_model_model = rwp, observation_model = obs_model,
     pos_shift = 1e-6)
-@time chn = sample(model,
+
+chn = sample(model,
```

Comment:
Not for this PR but I assume we are going to make some kind of EpiAware specific sample here that users pathfinder and then NUTs with our tested defaults here?

Reply:
Yes, everytime I improve something this script acts as end-to-end testing of the inference on generated data